insights/parsers/engine_config.py

```diff
@@ -0,0 +1,204 @@
+"""engine_config - command ``engine-config --all``
+===============================================
+
+This module provides access to ovirt-engine configuration parameters
+by parsing output of command ``engine-config --all``.
+"""
+from collections import namedtuple
+from operator import itemgetter
+
+from .. import Parser, parser, get_active_lines
+from insights.specs import Specs
+
+
+@parser(Specs.engine_config_all)
+class EngineConfigAll(Parser):
+    """Parsing output of command ``engine-config --all``
+
+    The parser tries its best to get `value` & `version` for `keyword`
+    specified. It ignores keywords where is fails. At the moment it
+    works well for output which has `keyword`, `value` & `version` in
+    a single line. It skip(rather fails) any `keyword` which has
+    multi-line output.
+
+    Typical output of ``engine-config --all`` command is::
+
+        MaxRerunVmOnVdsCount: 3 version: general
+        MaxStorageVdsTimeoutCheckSec: 30 version: general
+        ClusterRequiredRngSourcesDefault:  version: 3.6
+        ClusterRequiredRngSourcesDefault:  version: 4.0
+        ClusterRequiredRngSourcesDefault:  version: 4.1
+        HotUnplugCpuSupported: {"x86":"false","ppc":"false"} version: 3.6
+        HotUnplugCpuSupported: {"x86":"false","ppc":"false"} version: 4.0
+        HotUnplugCpuSupported: {"x86":"true","ppc":"true"} version: 4.1
+
+
+    Examples::
```

Comment:
@psachin because `Examples` is a recognized header it should only have 1 colon, if you look at the generated documentation you can see the problem.  Line 24 above it correct because you are designating the following indented block as a literal block.  Treat `Examples` like `Parameters`, `Attributes`, `Raises` and etc.

Reply:
@bfahr You are right. I see the problem in highlighting. I wondered why it display `Examples` header in different way. Thanks for pointing out.